developers_chamber/qa/checks.py

```diff
@@ -0,0 +1,78 @@
+import re
+
+from .base import QACheck, QAError
+
+
+class MissingMigrationsQACheck(QACheck):
+    """
+    Checks that make migrations command does not generate new migrations.
+    """
+    name = 'Check missing migrations'
+
+    def _run_check(self):
+        output = self._run_command('pydev make-migrations --dry-run')
+        last_line = output.split('\n')[-1]
+        if last_line != 'No changes detected':
+            raise QAError('Found missing migration(s)!', re.findall(r'\x1b.*?Migrations.+$', output, re.DOTALL)[0])
+
+
+class MigrationFilenamesQACheck(QACheck):
+    """
+    Checks that new migrations introduced in the branch have correct names.
+    """
+    name = 'Check migration filenames'
+
+    def _is_migration_file_with_wrong_name(self, path):
+        match = re.search(r'migrations\/([^\/]+)\.py$', path)
+        return bool(match and not re.search(r'^[0-9]{4}_migration$', match.group(1)))
+
+    def _run_check(self):
+        wrong_name_files = []
+        for diff in self._get_diffs():
+            if diff.new_file and self._is_migration_file_with_wrong_name(diff.b_path):
+                wrong_name_files.append(diff.b_path)
+
+        if wrong_name_files:
+            raise QAError('Found wrongly named migration file(s):', '\n'.join(wrong_name_files))
+
+
+class MissingTranslationsQACheck(QACheck):
+    """
+    Checks that generating translations does not introduce any changes.
+    """
+    name = 'Check missing translations'
+
+    def _is_translation_file(self, path):
+        return bool(re.search(r'django\.po$', path))
+
+    def _run_check(self):
+        self._run_command('pydev make-msgs')
```

Comment:
And I forgot about this. Have you considered adding an option to config which would set what command/alias should be used for generating messages/migrations? This isn't very reusable unless everyone would set the same aliases as we do.

Reply:
That looks like a viable solution, thanks. The point is, that these commands must be identical to those run on local machine, otherwise possible discrepancies would make fixing problems a nightmare. Your solution complies with that, luckily.